s3fs/tests/test_s3fs.py

```diff
@@ -256,7 +256,8 @@ def test_info(s3):
     linfo = s3.ls(a, detail=True)[0]
     assert abs(info.pop("LastModified") - linfo.pop("LastModified")).seconds < 1
     info.pop("VersionId")
-    info.pop("ContentType")
+    for key in S3File.additional_info_keys:
+        info.pop(key, None)
```

Comment:
I guess at this point, you really should test that you can a) set the MIME of an upload, b) append to that upload c) recover the original MIME. 
What happens if in b) you specify a different MIME? Does it depend on whether the file is smaller or larger than one binary block?